dask/dataframe/tests/test_dataframe.py

```diff
@@ -4429,3 +4429,37 @@ def test_fuse_roots():
     res = ddf1.where(ddf2)
     hlg = fuse_roots(res.__dask_graph__(), keys=res.__dask_keys__())
     hlg.validate()
+
+
+@pytest.mark.skipif(PANDAS_VERSION < "1.0.0", reason="attrs introduced in 1.0.0")
```

Comment:
```suggestion
@pytest.mark.skipif(not dd._compat.PANDAS_GT_100, reason="attrs intro]duced in 1.0.0")
```

Here and in the other tests.